stdlib/public/Concurrency/Task.swift

```diff
@@ -143,57 +159,58 @@ extension Task {
 
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 extension Task {
-  /// A task handle refers to an in-flight `Task`,
-  /// allowing for potentially awaiting for its result or Cancelling it.
+  /// An affordance to interact with an active task.
+  ///
+  /// You can use a task's handle to wait for its result or cancel the task.
   ///
-  /// It is not a programming error to drop a handle without awaiting or cancelling it,
-  /// i.e. the task will run regardless of the handle still being present or not.
-  /// Dropping a handle however means losing the ability to await on the task's result
-  /// and losing the ability to cancel it.
+  /// It's not a programming error to discard a task's handle without awaiting or canceling the task.
```

Comment:
Small side question, just to educate myself; so guidelines are to actually use "it's"? All my life in technical writing i've been avoiding it :) Is it because it's shorter/easier to read?

Reply:
I'm not sure the exact reason, but yes that's the style used here.  For documentation, we follow both [Apple Style Guide](https://help.apple.com/applestyleguide/) and style guidance from DevPubs that's specific to reference.  ASG's entry for *contractions* begins:

> As part of Apple’s informal voice, contractions are used and recommended throughout most documentation, interface text, and marketing copy. Keep localization in mind when deciding how or whether to use contractions.